pokeapp/src/components/PokemonCard.js

```diff
@@ -17,47 +25,66 @@ const PokemonCard = ({ name, id, types, picture, pokemon, setSelectedPokemon })
     width: `100%`,
     height: `75%`,
     borderRadius: `10px`,
-  }
+  };
   const pokemonPicker = (pokemon) => {
     setSelectedPokemon(pokemon);
   };
 
+  const myPokePicker = (pokemon) => {
+    console.log("hello");
```

Comment:
You naughty boy. Remove this `console.log`!

Reply:
lmao. Good looking out my guy!